src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
The class name 'IsoscelesTrapezoid' is appropriate as it is a noun. However, ensure that this class is only responsible for properties and behaviors that are relevant to an isosceles trapezoid to adhere to the single responsibility principle.